Sources/GRPCCore/GRPCServer.swift

```diff
@@ -194,152 +193,18 @@ public struct GRPCServer: Sendable {
       self.state.store(.stopped, ordering: .sequentiallyConsistent)
     }
 
-    if self.transports.isEmpty {
-      throw RuntimeError(
-        code: .noTransportsConfigured,
-        message: """
-          Can't start server, no transports are configured. You must add at least one transport \
-          to the server before calling 'run()'.
-          """
-      )
-    }
-
-    var listeners: [RPCAsyncSequence<Stream>] = []
-    listeners.reserveCapacity(self.transports.count)
-
-    for transport in self.transports {
-      do {
-        let listener = try await transport.listen()
-        listeners.append(listener)
-      } catch let cause {
-        // Failed to start, so start stopping.
-        self.state.store(.stopping, ordering: .sequentiallyConsistent)
-        // Some listeners may have started and have streams which need closing.
-        await self.rejectRequests(listeners)
+    self.state.store(.running, ordering: .sequentiallyConsistent)
```

Comment:
We effectively go from `starting` to `running` immediately which we can simplify by removing the `starting` state.